src/styles.css

```diff
@@ -1 +1,4 @@
 /* You can add global styles to this file, and also import other style files */
+
+html, body { height: 100%; }
```

Comment:
Don't change the styles.css. For now, keep them on a component basis within your [component].css. If it shows that we do need to incorporate this as a global style, we can alter it then